src/components/global/footer.astro

```diff
@@ -93,7 +93,7 @@
         class="flex sm:flex-row flex-col gap-10 items-center justify-between py-[30px] pb-5 px-0 border-t-[#303030] border-t-[1px] mt-8 border-solid"
       >
         <div class="[&>p]:text-[#404040] font-normal">
-          <p>© 2024. All rights reserved.</p>
+          <p>© {new Date().getFullYear()}. All rights reserved.</p>
```

Comment:
Since we do not suddenly stop hosting content from 2024, this should probably be a date range.